contract/assembly/model.ts

```diff
@@ -1,7 +1,7 @@
 // @nearfile
 import { storage } from 'near-sdk-as';
 
-const PROTOCOL_VERSION = "16";
+const PROTOCOL_VERSION = "17";
```

Comment:
Below need `@nearBindgen` decorator if you want to remove compiler warning.

Reply:
Sorry just realized I commented on the wrong line it should be the class declaration.